wrc/codegen/cghtml.py

```diff
@@ -90,10 +90,13 @@ class WCADocumentHtml(CGDocument):
     ''' Emit html formatted to fit in the WCA website.  '''
     name = "HTML"
 
-    def __init__(self, versionhash, language, pdf):
+    def __init__(self, versionhash, language, pdf, merged=False):
         super(WCADocumentHtml, self).__init__(str)
         self.regset = set()
-        self.urls = {'regulations': './', 'guidelines': './guidelines.html', 'pdf': pdf}
+        if merged:
+            self.urls = {'regulations': './full', 'guidelines': './full', 'pdf': pdf}
+        else:
+            self.urls = {'regulations': './', 'guidelines': './guidelines.html', 'pdf': pdf}
```

Comment:
I know that adding an additional default parameter to the constructor might not be an ideal option, but I think it's pretty straightforward and it shouldn't break anything.

Reply:
I agree and am in favor of backwards compatibility here :smile: